.github/workflows/test.yml

```diff
@@ -1,4 +1,7 @@
-on: push
+on:
+  push:
```

Comment:
I don't know YAML, should this be `push`?

Reply:
I don't know YAML either. But I think it's working as expected. Do you have an example of where it's not?